cli/pcluster/config/validators.py

```diff
@@ -342,7 +342,8 @@ def dcv_enabled_validator(param_key, param_value, pcluster_config):
     cluster_section = pcluster_config.get_section("cluster")
     if param_value == "master":
 
-        allowed_oses = get_supported_dcv_os()
+        architecture = cluster_section.get_param_value("architecture")  # x86_64 OR arm64
```

Comment:
Can you remove the comment at the end of this line? It would be very easy to forget to update this in the (unlikely) event we added support for another architecture.

Reply:
Yeah! Thanks for your reminding.